Packs/Cyberpion/Integrations/Cyberpion/Cyberpion.py

```diff
@@ -0,0 +1,530 @@
+import demistomock as demisto  # noqa: F401
+from typing import Dict, Tuple, List
+from datetime import timezone
+
+from CommonServerPython import *
+
+"""Cyberpion Integration for Cortex XSOAR (aka Demisto)
+"""
+
+''' IMPORTS '''
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%d %H:%M:%S.%f %Z'
+
+MAX_PAGES_TO_GRAB = 5
```

Comment:
Why is this hard coded and not provided as command argument that the user can control?

Reply:
This is a date format our server use. We use this format to ask action items from server (by date created).

Reply:
the comment was on `MAX_PAGES_TO_GRAB`, not the date format

Reply:
please note this comment